src/main/scala/com/databricks/spark/sql/perf/mllib/fpm/ItemSetGenerator.scala

```diff
@@ -0,0 +1,60 @@
+package com.databricks.spark.sql.perf.mllib.fpm
+
+import com.databricks.spark.sql.perf.mllib.data.DataGenUtil
+
+import scala.collection.mutable.ArrayBuffer
```

Comment:
style: please always organize imports!  It's worth modifying IntelliJ settings to do this for you: IntelliJ -> Preferences -> Editor -> Code Style -> Scala -> "Imports" tab